Source/WebKit/WebProcess/WebCoreSupport/WebNotificationClient.cpp

```diff
@@ -54,7 +54,7 @@ bool WebNotificationClient::show(ScriptExecutionContext& context, NotificationDa
 {
     bool result;
     callOnMainRunLoopAndWait([&result, notification = WTFMove(notification).isolatedCopy(), resources = WTFMove(resources), page = m_page, contextIdentifier = context.identifier(), callbackIdentifier = context.addNotificationCallback(WTFMove(callback))]() mutable {
-        result = WebProcess::singleton().supplement<WebNotificationManager>()->show(WTFMove(notification), WTFMove(resources), page.get(), [contextIdentifier, callbackIdentifier] {
+        result = WebProcess::singleton().supplement<WebNotificationManager>()->show(WTFMove(notification), WTFMove(resources), RefPtr { page.get() }.get(), [contextIdentifier, callbackIdentifier] {
```

Comment:
Would doing `page = RefPtr { m_page.get() }` on the line above be better ?

Reply:
I don't think so, not at least in terms of semantics.